test/integration/sonobuoy_integration_test.go

```diff
@@ -49,14 +53,21 @@ func findSonobuoyCLI() (string, error) {
 
 // runSonobuoyCommandWithContext runs the Sonobuoy CLI with the given context and arguments.
 // It returns any encountered error and the stdout and stderr from the command execution.
-func runSonobuoyCommandWithContext(ctx context.Context, t *testing.T, args string) (bytes.Buffer, error) {
+func runSonobuoyCommandWithContext(ctx context.Context, t *testing.T, args string, env ...string) (bytes.Buffer, error) {
 	var combinedOutput bytes.Buffer
 
 	command := exec.CommandContext(ctx, sonobuoy, strings.Fields(args)...)
 	command.Stdout = &combinedOutput
 	command.Stderr = &combinedOutput
 
-	t.Logf("Running %q\n", command.String())
+	// Test with features on so we can get more feedback. Low risk that
+	// this will hide default behavior but in that case we may just make the
+	// experimental feature the default then.
+	command.Env = []string{"SONOBUOY_ALL_FEATURES=true", "KUBECONFIG=" + os.Getenv("KUBECONFIG"), "HOME=" + testHome}
```

Comment:
This seems also to be a good exercise in realizing the requirements of sonobuoy/our tests.

Had failed to explicitly copy kubeconfig here and so test run failed to connect to cluster.

FYI the reason we have to do all this is because if you _dont_ set the command env then it uses the current os env. Since we are setting it for each command, we have to set everything, we aren't just adding to the os.env